src/Nethermind/Nethermind.Trie/RustVerkleLib.cs

```diff
@@ -37,7 +37,7 @@ static RustVerkleLib()
         }
         
         [DllImport("rust_verkle")]
-        private static extern IntPtr verkle_trie_new();
+        private static extern IntPtr verkle_trie_new(byte databaseScheme, byte commitScheme);
```

Comment:
define a enum specifying the different values for databaseScheme and commitScheme

Reply:
something like this
https://github.com/NethermindEth/nethermind/blob/verkle-trie/src/Nethermind/Nethermind.State/VerkleStateProvider.cs#L43

Reply:
It can be also an actual enum based on byte type